test/functions/bradc/arrayFormals/formalArrArgErrorLoc.chpl

```diff
@@ -0,0 +1,7 @@
+proc foo(X: [1..3] real) {
+  writeln("In foo!");
+}
+
+var A: [1..10] real;
+
+foo(A);
```

Comment:
Maybe skip this test for `--fast`, otherwise I think you'll have a .bad file mismatch